src/Forms/Validation.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { createContext, useState, FC } from "react";
+import Schema, { Rules } from "async-validator";
+
+export type FieldsObject = { [key: string]: unknown };
+
+export type Validator = (
+  fields: FieldsObject,
+  rulesOrID: Rules | string
+) => boolean;
+
+const validateFn: Validator = (_fields, _rulesOrID) => false;
```

Comment:
I don't like this `rulesOrID` tbh so please take a closer look if I missed something

Reply:
you mean the name ?

Reply:
no I mean I called it like this because that's what I THINK it means but not really sure

Reply:
yeah they are pretty different if we see their functionality